LICENSE.txt

```diff
@@ -202,7 +202,7 @@
    limitations under the License.
 
 --------------------------------------------------------------------------------
-This product includes code from Netty 4.1.117.Final:
+This product includes code from Netty 4.1.119.Final:
```

Comment:
Do we need to update this?

I think that we should use the version when we copied.

Reply:
Yes that's correct. My bad: my script was too aggressive 😀
Let me fix that.

Reply:
Wow! You have a script for this!? Could you share it?

Reply:
Oh yes. I will create a PR to add the script and maybe include in CI.